bin/setup

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env ruby
 require 'pathname'
 
+def system(*args)
+  puts args.first
+  super
+end
```

Comment:
ok, why not. Then, it's like `set -x` in bash and the scope is only for this script so we are not doing anything wild here.